File: src/model/statement/CompStmt.java
package model.statement;
import model.ProgState;
import utils.*;
public class CompStmt implements Statement{
	
	private Statement first;
	private Statement second;
	
	public CompStmt(Statement first, Statement second){
		this.first = first;
		this.second = second;
	}
	
	@Override
	public ProgState execute(ProgState p) {
		IExeStack<Statement> stack1 = p.getExeStack();
		stack1.push(second);
		stack1.push(first);
		return null;
	}
	
	public String toString(){
		return "{" + first + ";" + second + "}";
	}
}

File: ToyLanguageInterpretorC#/ToyLanguageC/model/CompStmt.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ToyLanguageC.utils;

namespace ToyLanguageC.model
{
    class CompStmt : Statement
    {
        Statement stm1;
        Statement stm2;

        public CompStmt(Statement stm1,Statement stm2)
        {
            this.stm1 = stm1;
            this.stm2 = stm2;
        }

        public ProgState execute(ProgState state)
        {
            ExeStack<Statement> stack = state.getExeStack;
            stack.push(stm2);
            stack.push(stm1);
            return state;
        }

        public string toString()
        {
            return stm1.toString() + "; " + stm2.toString();
        }
    }
}
